### PR TITLE
Creates array to clean meta-data

### PR DESCRIPTION
**Pull request by proseLA**
removes `*` in addition to `"` from meta-data.  also allows to remove any character from meta-data, by just adding that character to the array.

### CODE REVIEW

File: includes/modules/meta_tags.php [hunk 1]
```diff
@@ -22,6 +22,8 @@
 } else {
   define('TAGLINE', '');
 }
+// array to remove specific characters from meta-data
+$bad_characters = array('"','*');
```
Comment (drbyte) [inline includes/modules/meta_tags.php]: Can you move this initialization of `$bad_characters` up to line 16 (replace the comment line of "this should be the first line" with your lines 25-26.

Then add `$bad_characters` to the list of params passed to the notifier on line 17 above. This way people can extend the list by using an observer class.

**Comment by proseLA** [comment]
ok.  i think i made those things happen.... i hope i understood it correctly as i did delete the comment line of "this should be the first line..."

**Comment by drbyte** [comment]
👍 
That's exactly what I was looking for.

**Comment by zcwilt** [comment]
:+1:

**Pull request merged**